webui/react/src/pages/SignOut.tsx

```diff
@@ -35,7 +34,6 @@ const SignOut: React.FC = () => {
 
     if (!isSigningOut) signOut();
 
```

Comment:
@hkang1 do we actually want to cancel the request here? :thinking:

Reply:
@hamidzr good question, do we have the ability to cancel requests via our new API?

Reply:
we talked about this in UI architecture: https://developer.mozilla.org/en-US/docs/Web/API/AbortController/abort